setup.py

```diff
@@ -9,20 +9,22 @@
 
 def get_version():
     basedir = os.path.dirname(__file__)
-    with open(os.path.join(basedir, 'rq/version.py')) as f:
-        locals = {}
-        exec(f.read(), locals)
-        return locals['VERSION']
-    raise RuntimeError('No version info found.')
+    try:
+        with open(os.path.join(basedir, 'rq/version.py')) as f:
+            locals = {}
+            exec(f.read(), locals)
+            return locals['VERSION']
+    except FileNotFoundError:
+        raise RuntimeError('No version info found.')
```

Comment:
Now it will properly throw `RuntimeError`

Reply:
Why is it necessary to throw a `RuntimeError` instead of a `FileNotFoundError`?